cylc/flow/task_pool.py

```diff
@@ -780,6 +789,8 @@ def remove(self, itask, reason=""):
                 del self.hidden_pool[itask.point]
             LOG.debug(f"[{itask}] {msg}")
             self.task_queue_mgr.remove_task(itask)
+            self.tasks_to_hold.discard((itask.tdef.name, itask.point))
+            LOG.info(f"Releasing held task {itask}")
```

Comment:
This message does kinda sound like we should expect the task to run.

Perhaps "Removing held task"?

Reply:
Good spot, pretty sure I meant to type "Removing" there!